Frigate Camera Notifications/Stable

```diff
@@ -730,8 +746,10 @@ action:
                               attachment:
                                 url: "/api/frigate/notifications/{{id}}/{{attachment}}.jpg"
                               push:
-                                sound: "{{sound}}"
-                                interruption-level: "{{ iif(critical, 'critical', 'active') }}"
+                                sound:
+                                  name: "{{sound}}"
+                                  volume: "{{ iif(sound == 'none', 0, volume) }}"
```

Comment:
How do we handle the default sound if people just want that. potentially causing issues for people who just want to enter the camera name and device and get notifications

Reply:
"default" sound is already `default: default` and "default" volume is `default: 100`

So people who specify neither of the two have 100% volume of default sound, like now.

--
When "none" is selected, the sound is set to "0".

However, if nothing is selected, the volume is taken as volume: "{{ iif(sound == 'none', 0, volume) }}".
 
If the volume is '0', there will be no sound regardless of the selected option for the sound, it's like sound is 'none'.